src-vue/src/views/RepairView.vue

```diff
@@ -87,6 +88,16 @@ export default defineComponent({
             });
 
             let install_northstar_result = invoke("install_northstar_caller", { gamePath: game_install.game_path, northstarPackageName: ReleaseCanal.RELEASE });
+
+            const unlistenProgress = await appWindow.listen(
```

Comment:
What is this listener used for?

Reply:
It's never used, I just copy and pasted code together until it worked ^^"

Reply:
Addressed in a9180cfbecdd3affc860e52af2b43f740cfee371